core/src/test/scala/org/apache/spark/status/AppStatusStoreSuite.scala

```diff
@@ -115,6 +131,42 @@ class AppStatusStoreSuite extends SparkFunSuite {
     }
   }
 
+
+  test("SPARK-26260: task summary size for default metrics should be zero") {
```

Comment:
Couldn't you cover this in the test below by adding some "RUNNING" tasks (with and without metrics)?

Reply:
Yes, I have modified the test suite. Thanks